packages/block-library/src/image/style.scss

```diff
@@ -157,25 +157,48 @@
 	display: flex;
 	flex-direction: column;
 
+	img {
+		cursor: zoom-in;
+	}
+
 	button {
+		opacity: 0;
 		border: none;
-		background: none;
+		background: #1e1e1e;
```

Comment:
Not sure we should use a UI color for this button. It can probably just be black to begin with. In the future we might consider making the appearance consistent with other Buttons as defined by the active theme.

Reply:
Agree with black.